packages/react-components/src/components/VideoTile.tsx

```diff
@@ -403,7 +412,7 @@ const participantStateStringTrampoline = (props: VideoTileProps, locale: Compone
   const strings = { ...locale.strings.videoTile, ...props.strings };
   /* @conditional-compile-remove(one-to-n-calling) */
   /* @conditional-compile-remove(PSTN-calls) */
-  return props.participantState === 'Idle' || props.participantState === 'Connecting'
+  return props.participantState === 'Connecting'
```

Comment:
with the change to the selector, can we remove this?

Reply:
I assume without the change the video gallery will still say "Calling.." for teams interop participants

Reply:
Done! just waiting for snapshots now